src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,155 @@
+package duke;
+
+import duke.exception.DukeException;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class Storage {
+    /** The filePath to load and save the tasks. **/
+    private String filePath;
+
+    /**
+     * A public constructor to initialized the Storage.
+     *
+     * @param filePath The given filePath.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * A method to load the task from the file, and add them to the data structure
+     * in TaskList
+     *
+     * @return An ArrayList of Task, the list of tasks loaded from the file.
+     * @throws DukeException
+     */
+    public ArrayList<Task> loadTaskList() throws DukeException{
+        File f;
+        Scanner sc;
+        try {
+            f = new File(filePath);
+            /** If folder data does not exist, mkdir. **/
+            if (!f.getParentFile().exists()) {
+                f.getParentFile().mkdir();
+            }
+            /** If file taskList.txt does not exist, create it. **/
+            if (!f.exists()) {
+                f.createNewFile();
+            }
+            sc = new Scanner(f);
+
+            int n = sc.nextInt();
+            //        System.out.println("total number of tasks loaded from file = " + n);
```

Comment:
Should this line be deleted since it is not used anymore? I notice a few other occurrences of this in this file as well.